src/components/DonateButton.jsx

```diff
@@ -263,12 +263,12 @@ class DonateButton extends Component {
                   onChange={(name, value) => this.setState({ mewAmount: value })}
                   placeholder="10"
                   validations={{
-                    lessThan: wallet.getBalance() - 0.5,
-                    greaterThan: 0.00000000009,
+                    lessThan: wallet.getBalance() + (gas * 1000000000),
+                    greaterEqualThan: 0.1,
```

Comment:
I don't see a reason to have a minimum value of .1 eth. That was $80 the other day. If we want to have a minimum value, maybe it should be the gas used in the tx? @vojtechsimetka what do you think?